Weather/Models/DailyWeatherData.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DailyWeatherData.swift
+//  Weather
+//
+//  Created by Alex on 31.07.22.
+//
+
+import Foundation
+
+struct DailyWeatherData: Codable, Identifiable {
+    var dt: Int
+    var temp: TemperatureData
+    var weather: [WeatherDetail]
```

Comment:
I see that you use `DailyWeatherData` not only for the network request. In this case I would suggest you using correct naming not the one on the API: `date`, `temperature`, `weatherDetails`

Reply:
Fixed.)